BREAKING.md

```diff
@@ -24,6 +25,9 @@ Previously the `@fluidframework/common-utils` package exposed a `setInsecureCont
 ### Changes to PureDataObjectFactory
 - The `createDataObject` in `PureDataObjectFactory` has a mandatory `existing` parameter to differentiate creating vs loading.
 
+### `webpack-fluid-loader` package name changed
+The `webpack-fluid-loader` utility was previously available from a package named `@fluidframework/webpack-fluid-loader`.  However, since it is intended as an example tool not for use in production, it is now available under the tools scope `@fluid-tools/webpack-fluid-loader`.
+
```

Comment:
Possibly stale prose?  Describing the tool as an example sounds as if we expect developers to learn from and modify the tool rather than use it as-is.

Perhaps: "However, since it is a development tool intended for developer convenience rather, it is now available under the tools scope `@fluid-tools/webpack-fluid-loader`."

Reply:
Updated to be more clear.